CMakeLists.txt

```diff
@@ -189,3 +194,5 @@ endif()
 if(BUILD_DOCS)
   add_subdirectory(docs)
 endif()
+
+include (${CMAKE_CURRENT_SOURCE_DIR}/CPackConfig.cmake)
```

Comment:
No need to prefix with `${CMAKE_CURRENT_SOURCE_DIR}`

Reply:
adjusted it